eessi-2023.06-eb-4.8.1-system.yml

```diff
@@ -4,3 +4,6 @@ easyconfigs:
   - ReFrame-4.3.3.eb:
       options:
         from-pr: 18851
+  - EasyBuild-4.8.2.eb:
+      options:
+        from-pr: 19101
```

Comment:
I'm not sure how this worked before, but the correct PR is actually [this one](https://github.com/easybuilders/easybuild-easyconfigs/pull/19105), i.e. `19105`.